fastlane/lib/fastlane/fastfile_parser.rb

```diff
@@ -0,0 +1,324 @@
+require 'parser'
+require 'parser/current'
+require 'pp'
+require "terminal-table"
+module Fastlane
+  class FastfileParser
+    attr_accessor :original_action
+
+    def to_s
+      "REMOVE_IT"
```

Comment:
thats a placeholder for variables 👍  like if user has something like this:


```
action(param: "#{some_ruby_thing} asdfg")
```

those will be replaced by a FastfileParser class - and via `to_s`  replaced with "REMOVE_IT".